pkg/registry/apis/scope/register.go

```diff
@@ -29,7 +29,7 @@ func NewScopeAPIBuilder() *ScopeAPIBuilder {
 }
 
 func RegisterAPIService(features featuremgmt.FeatureToggles, apiregistration builder.APIRegistrar, reg prometheus.Registerer) *ScopeAPIBuilder {
-	if !features.IsEnabledGlobally(featuremgmt.FlagGrafanaAPIServerWithExperimentalAPIs) {
+	if !features.IsEnabledGlobally(featuremgmt.FlagScopeApi) {
```

Comment:
```suggestion
	if !(features.IsEnabledGlobally(featuremgmt.FlagScopeApi) ||
	features.IsEnabledGlobally(featuremgmt.FlagGrafanaAPIServerWithExperimentalAPIs)) {
```
maybe -- then it will keep working in the existing dev environments

Reply:
good point!